src/js/components/Carousel.js

```diff
@@ -224,7 +224,13 @@ export default class Carousel extends Component {
     const { children } = this.props;
     const { activeIndex } = this.state;
     const numSlides = children.length;
-    const index = (activeIndex + numSlides - 1) % numSlides;
```

Comment:
Thx @alansouzati.   My continuing problem, however, is the coverage fail. Can you suggest fix?